### DISCUSSION TITLE
My Chias disappeared from blockchain

### DISCUSSION THREAD

**Discussion by Lightning06200** [Chia-Network/chia-blockchain#5020] [Support]
My chia disappeared from blockchain

Hello guys Maybe someone can help me. Few days ago I managed to get some chias from 2 different faucets. They didn’t appeared in my GUI wallet but when I checked my wallet address in the blockchain they appeared clearly. But today when I checked again my address on the blockchain the transactions disappeared (and still no chias in the GUI) and shows no transactions. I really don’t understand. I have 200+ more plots farming and still 0 chias. I’m starting to think that Chia is just buggy af. Or maybe I’m doing something wrong. Someone can help me?

**Comment by moshenskyDV** [comment]
Could you check your address on chiaexplorer: [https://www.chiaexplorer.com/blockchain/search](https://www.chiaexplorer.com/blockchain/search)
Any operations with it?

**Reply by Lightning06200** [reply]
I've already checked 3 days ago and I could see 2 transactions but today when I put my wallet address in the explorer it says that the search returned no results. When I go to try again for the faucet it says ''already sent''

**Reply by moshenskyDV** [reply]
Did you generate block on version < 1.1.5?

**Reply by Lightning06200** [reply]
How can I check that?

**Reply by moshenskyDV** [reply]
Did you update your desktop app after block generating?
Press Help -> About Chia Blockchain

If you generated it on version 1.1.4 or lower it could be rejected by 1.1.5 nodes (check 1.1.5 release), developers fixed big mess, and some of blocks from old nodes goes rejected now. If you still haven't updated to 1.1.5 I strongly recommend to do this to avoid rejected blocks in future.

**Reply by Lightning06200** [reply]
Yes I did the update to 1.1.5. Is there a way to recover my chias? Thank you for your kind help.

**Reply by moshenskyDV** [reply]
Did you generate block on 1.1.5 or not? Try to remember what version was on your PC at the moment of time?
If it was 1.1.4 - then probably you generated "negative" block and no way to return it.
If it was 1.1.5 - then it's strange.

**Reply by Lightning06200** [reply]
It was on 1.1.5 I think because I updated it immediately because of the problems in 1.1.4

**Reply by Lightning06200** [reply]
For the official faucet it was on 1.1.5 and for unofficial it was on 1.1.4 so 2 transactions disappeared

**Reply by moshenskyDV** [reply]
Oh sorry, I just double checked the message. I thought you got reward by generating block.
Than forget my last 3 messages. It's out of scope of your question.

BTW: I tried to use faucet and I haven't got any coins as well  ;(

**Reply by Lightning06200** [reply]
No problem my friend but it's strange that they appeared in the blockchain and now they completely disappeared...

**Comment by lvcivs** [comment]
Well, the transactions can't just disappear. Transactions on the blockchain are irreversible. So it sounds to me like chiaexplorer was not working properly or you did something wrong when searching. Are you sure you checked the right wallet address? Have you tried checking the same wallet address also on chiastatus.com , for comparison?

Also you may want to look into why the wallet in your GUI does not show the right balance. Maybe a syncing problem?

**Comment by Lightning06200** [comment]
> Well, the transactions can't just disappear. Transactions on the blockchain are irreversible. So it sounds to me like chiaexplorer was not working properly or you did something wrong when searching. Are you sure you checked the right wallet address? Have you tried checking the same wallet address also on chiastatus.com , for comparison?
> 
> 
> 
> Also you may want to look into why the wallet in your GUI does not show the right balance. Maybe a syncing problem?

Hello, I tried on chiastatus.com and my 2 transactions appeared but shows zero chias (probably because of the very low amount for the faucet). However the GUI don't show them and I'm synced with the network (around 50 connections). Do you have a clue for this problem?

**Reply by lvcivs** [reply]
The only thing I can think of is that your wallet may have gotten corrupted? You could try deleting the wallet db and syncing it from zero again.

**Reply by Lightning06200** [reply]
Thank you I will definitely try that and I'll let you know.

**Reply by nenedelnorte** [reply]
https://github.com/Chia-Network/chia-blockchain/discussions/5101

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
